### ISSUE TITLE
Generative opera

### ISSUE THREAD

**Issue by nhthn** [NaNoGenMo/2022#17]
Not a novel since it definitely won't reach the word count, but I feel it's in the spirit of the challenge to make at least two hours of music. I plan on having a mixture of algorithmic and handwritten elements.

For vocals I will be using [OddVoices](https://oddvoices.org/), a free and open source singing synthesizer for English that I developed. For instrumentation I plan on synthesizing everything in SuperCollider.

**Comment by hugovk** [comment]
Indeed, there has been a NaOpGenMo (National Opera Generation Month) in [2014](https://github.com/cpressey/NaOpGenMo) and [2015](https://github.com/cpressey/NaOpGenMo-2015)! But please do continue!

cc @cpressey

**Comment by cpressey** [comment]
🧑‍🎤 !
